docs/_includes/projects/style/perinstanceexpressions/my-element.js

```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit-element';
+
+const perClassStyle = html`
```

Comment:
WDYT about adding a comment here like:

`// Note per-class styles should almost always be defined using the static styles array instead of with a <style> tag.`

So that people who jumped straight here don't think this is recommended.